src/GrampsJs.js

```diff
@@ -796,6 +822,102 @@ export class GrampsJs extends LitElement {
     this._showToast(message)
   }
 
+  async _handleOIDCCallback() {
```

Comment:
Please move all of the OIDC logic to a separate file instead of putting it into a method. The GrampsJs object is already a monster and hardly maintainable.